README.md

```diff
@@ -215,15 +218,21 @@ Inspired by [awesome-python](https://github.com/vinta/awesome-python).
 
 *Tools for managing model lifecycle (tracking experiments, parameters and metrics).*
 
+* [AIF360](https://github.com/Trusted-AI/AIF360) - A comprehensive set of fairness metrics for datasets and machine learning models.
 * [Aim](https://github.com/aimhubio/aim) - A super-easy way to record, search and compare 1000s of ML training runs.
 * [Comet](https://github.com/comet-ml) - Track your datasets, code changes, experimentation history, and models.
+* [Fairlearn](https://github.com/fairlearn/fairlearn) - A Python package to assess and improve fairness of machine learning models.
 * [Guild AI](https://guild.ai/) - Open source experiment tracking, pipeline automation, and hyperparameter tuning.
 * [Keepsake](https://github.com/replicate/keepsake) - Version control for machine learning with support to Amazon S3 and Google Cloud Storage.
+* [MinIO](https://github.com/minio/minio) - Pioneering high-performance, Kubernetes-native object storage.
```

Comment:
I don't think `MinIO` should be included as a "Model Lifecycle" tool. As far as I know it is a general object storage (like S3, GCS), sure it can be used as a storage layer for other applications like MLflow, but the tool itself is not focused on ML.

Reply:
That's okay. I can delete it from the list.